nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -301,6 +301,14 @@ def export_model(self, model_path, calibration_path=None, onnx_path=None, input_
                 calibration_config[name]['tracked_min_weight'] = -val
                 calibration_config[name]['tracked_qmin_weight'] = -127
                 calibration_config[name]['tracked_qmax_weight'] = 127
+                weight = module.weight
+                quantized_weight = self._quantize(weight,
```

Comment:
why we need `_quantize` again?

Reply:
+1

Reply:
We have to make sure that the type of `module.weight` is `torch.nn.Parameter` in ut [test_compressor_torch](https://github.com/microsoft/nni/blob/master/test/ut/sdk/test_compressor_torch.py#L303). However, this pr register `weight` as buffer in quantizers.py line256 to fix assign weight problem. So we have to convert it as Parameter during `export_model`. And I choose to convert it in this way since it will not influence the correctness.

Reply:
need add comments above this line to explain